x/ibc/02-client/keeper/keeper.go

```diff
@@ -168,11 +173,35 @@ func (k Keeper) SetCommitter(ctx sdk.Context, clientID string, height uint64, co
 	store.Set(types.KeyCommitter(clientID, height), bz)
 }
 
+// GetConsensusState introspects the past historical info at a given height and
+// returns the expected consensus state at that height.
+func (k Keeper) GetConsensusState(ctx sdk.Context, height int64) (exported.ConsensusState, bool) {
+	histInfo, found := k.stakingKeeper.GetHistoricalInfo(ctx, height)
+	if !found {
+		return nil, false
+	}
+
+	// query the following historical info to get the next validator set
+	nextHistInfo, found := k.stakingKeeper.GetHistoricalInfo(ctx, height+1)
+	if !found {
+		return nil, false
+	}
+
+	consensusState := tendermint.ConsensusState{
```

Comment:
We don't need this type here, this function is just used for retrieving a previously verified commitment root.